promise/src/components/search-bar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import searchIcon from './search.png';
+
+export const SearchBar = ({onInputChange, inputValue, placeholder,initializeSearch}) => {
+
+	const keyPressHandler = e => {
+		if (e.keyCode===13) {initializeSearch()}
```

Comment:
Remember to add a `debounce` mechanism when we have input changes that trigger network request.

For example, every character the user types, it would trigger a search. But what if the user has not finished? They want to type `encyclopedia` but we are hitting the server a lot to achieve this.

Debounce helps solve this problem by timing the space between each character inputted.